js/options.js

```diff
@@ -47,6 +48,21 @@
   };
 
   doc.addEventListener('DOMContentLoaded', function() {
+    var fontOpt = localStorage.getItem('font');
+    var fontEl = doc.querySelector('#font');
+    chrome.fontSettings.getFontList(function (fonts) {
+      var frag = doc.createDocumentFragment();
+      fonts.forEach(function (font) {
+        var opt = doc.createElement('option');
+        opt.textContent = font.displayName;
+        opt.value = font.fontId;
+        frag.appendChild(opt);
+      });
+      doc.querySelector('#font-system').appendChild(frag);
+      fontEl.value = fontOpt;
+      fontEl.dispatchEvent(new Event('change'));
```

Comment:
Unfortunately they are as `chrome.fontSettings.getFontList` is async, otherwise you get a race condition :car: